lib/iris/fileformats/um_cf_map.py

```diff
@@ -815,6 +817,18 @@
     'm04s06i001': CFName('sea_surface_wind_wave_significant_height', None, 'm'),
     }
 
+_STASHCODE_IMPLIED_HEIGHTS = {
```

Comment:
Do all of the "surface" type phenomenon deserve similar treatment to say that they have a height of 0? I don't propose removing the "surface" from the std name, just adding some extra metadata which should really be in the coordinate, not the phenomenon description...